ballerina/modules/wssec/encryption.bal

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
// WSO2 LLC. licenses this file to you under the Apache License,
```

Reply:
Fixed in [6f98be1](https://github.com/ballerina-platform/module-ballerina-soap/pull/103/commits/6f98be1c6f54b1b38af287f7550eca152b247812)